examples/purchaseTesterTypescript/.buckconfig

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
idk if this kinda file is common knowledge for RN developers (and maybe ignore me if so), but could we add a comment on what this does for us?